frontend/src/app/organizations/organizations.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { environment } from '../../environments/environment';
+import { Organization } from 'app/organization/organization.model';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class OrganizationsService {
+  private url = `${environment.apiRoot}/organizations/`;
+
+  constructor(private http: Http) { }
+
+  getOrganizations(): Observable<Organization[]> {
```

Comment:
I don't see any reason to have two separate services for single organization and for multiple of them - can You merge this `getOrganizations` into `OrganizationService` in `frontend/src/app/organization/organization.service.ts`?

Reply:
Method getOrganizations  was moved to organization.service.ts